ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -160,16 +168,16 @@ runDataDiffusion tracers
         Async.withAsync (cleanNetworkMutableState networkLocalState) $ \cleanLocalNetworkStateThread ->
 
           -- fork server for local clients
-          Async.withAsync (runLocalServer localSnocket networkLocalState) $ \_ ->
+          Async.withAsync (runLocalServer iocp networkLocalState) $ \_ ->
```

Comment:
That's probably not related to this PR, so you can ignore it: elsewhere we use `ioManager` rather than `iocp`.

Reply:
Yes, it is called `iocp` across the file.